static/styles/portico-signin.css

```diff
@@ -568,6 +568,7 @@ html {
     left: 5px;
 }
 
+.login-github,
```

Comment:
Is this meant to be `.login-mediawiki`?

Reply:
Nope. This adds a spacing between Github button and MediaWiki button. I know this looks confusing, but I went with this approach to mimic `.login-google`.